kotlin-mbedtls-metrics/src/main/kotlin/org/opencoap/ssl/transport/metrics/micrometer/DtlsServerMetricsCallbacks.kt

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2022-2023 kotlin-mbedtls contributors (https://github.com/open-coap/kotlin-mbedtls)
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.opencoap.ssl.transport.metrics.micrometer
+
+import io.micrometer.core.instrument.Counter
+import io.micrometer.core.instrument.Meter
+import io.micrometer.core.instrument.MeterRegistry
+import io.micrometer.core.instrument.config.MeterFilter
+import io.micrometer.core.instrument.distribution.DistributionStatisticConfig
+import org.opencoap.ssl.HelloVerifyRequired
+import org.opencoap.ssl.SslHandshakeContext
+import org.opencoap.ssl.SslSession
+import org.opencoap.ssl.transport.DtlsServer
+import java.net.InetSocketAddress
+import java.util.concurrent.TimeUnit
+
+class DtlsServerMetricsCallbacks(
+    private val registry: MeterRegistry,
+    metricsPrefix: String = "dtls.server",
+    private val distributionStatisticConfig: DistributionStatisticConfig =
+        DistributionStatisticConfig.Builder().percentiles(0.5, 0.9, 0.95, 0.99).build(),
```

Comment:
Could this default to nothing, and let the user choose percentiles?

Reply:
Just made it consistent with java-coap. And java-coap was made similar to ktor metrics.